README.md

```diff
@@ -43,8 +43,10 @@ Planned Models & Platforms (not implemented yet):
 Supported Stores
 ----------------
 
+<<<<<<< HEAD
```

Comment:
You might want to remove the conflict marker

Reply:
Jepp